Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,84 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            //Set variable
+            bool backtotop = true;
+            string option;
+
+            //Start While loop
+            while (backtotop == true)
+
+            {
+                backtotop = true;
+
+                //USer data input
+
+                Console.WriteLine("What is your first name?");
+                var firstname = Console.ReadLine();
+
+                Console.WriteLine("What is your second name?");
+                var secondname = Console.ReadLine();
+
+                Console.WriteLine("What is your age?");
+                var age = Console.ReadLine();
+
+                Console.WriteLine("What is your weight in kg?");
+                var weight = Console.ReadLine();
+
+                Console.WriteLine("What is your height in Meters?");
+                var height = Console.ReadLine();
+
+
+                //Convert vars
+                var weightsingle = Convert.ToSingle(weight);
+                var heightsingle = Convert.ToSingle(height);
+
+
+                //Calc for BMI
+                var Calc1 = weightsingle / heightsingle;
+                var BMI = Calc1 / heightsingle;
```

Comment:
What annoys me personally, is the usage of var here. It's not very intuitive what the types are when reading it. Maybe a bit because of ToSingle. Probably people disagree, so you don't have to change this but I felt like sharing my opinion.

Reply:
Changed vars to float.

Reply:
I don't see a change, did you push it?

Reply:
I changed them all to vars.